config/defaults.json

```diff
@@ -2,7 +2,7 @@
   "vault": {
     "port": 8200,
     "hostname": "localhost",
-    "token": "3e5f8293-eb83-37be-2ed3-03e4324c5e53"
+    "token":
```

Comment:
Will this still parse as valid JSON without an empty string for the value? Would it make more sense to just remove the `token` key entirely?